### DISCUSSION TITLE
Allow custom labels for lists

### DISCUSSION THREAD

**Discussion by hypeJunction** [keystonejs/keystone#6231] [Feature requests]
Either it's not described in documentation, or the feature is missing. It would be great to allow custom labels for lists in Admin sidebar navigation, as well as custom route slugs.

**Comment by raveling** [comment]
@hypeJunction take a look at:

https://keystonejs.com/docs/guides/custom-admin-ui-navigation
https://keystonejs.com/docs/guides/custom-admin-ui-pages

Does this meet your needs?

**Comment by dcousens** [comment]
Added in https://github.com/keystonejs/keystone/pull/7657
